src/Commands/AboutCommandGroup.cs

```diff
@@ -100,7 +100,7 @@ private async Task<Result> SendAboutBotAsync(IUser bot, Snowflake guildId, Cance
             .WithSmallTitle(string.Format(Messages.AboutBot, bot.Username), bot)
             .WithDescription(builder.ToString())
             .WithColour(ColorsList.Cyan)
-            .WithImageUrl("https://i.ibb.co/fS6wZhh/octobot-banner.png")
+            .WithImageUrl("https://github.com/TeamOctolings/Octobot/raw/master/docs/octobot-banner.png")
```

Comment:
No need for unnecessary redirects
```suggestion
            .WithImageUrl("https://raw.githubusercontent.com/TeamOctolings/Octobot/master/docs/octobot-banner.png")
```